package.json

```diff
@@ -73,7 +73,6 @@
   },
   "devDependencies": {
     "@babel/core": "^7.25.8",
-    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-transform-runtime": "^7.25.7",
```

Comment:
I believe we can get rid of more plugins: https://github.com/RedHatInsights/insights-inventory-frontend/commit/a587c601e03614e8714f3ce4e8d68698c1c71929

Reply:
Good point, I removed `@babel/plugin-syntax-dynamic-import` as well.